apps/hash-api/src/graph/ontology/primitive/entity-type.ts

```diff
@@ -128,7 +134,12 @@ export const updateEntityType: ImpureGraphFunction<
   const updateArguments: UpdateEntityTypeRequest = {
     actorId,
     typeToUpdate: entityTypeId,
-    schema,
+    schema: {
+      kind: "entityType",
+      $schema:
```

Comment:
Ridiculously nit-picky suggestion (feel free to ignore), would be nice to try and establish a consistent ordering for these fields. We've been going with `$schema`, `kind`, `$id`, `title`, ...

Only worth addressing if you're already going to touch these files, 100% not blocking

Reply:
I tried but obviously missed a few. I will go through the changed files again and apply the order.

Reply:
No need! Really not important